rush.json

```diff
@@ -488,6 +488,12 @@
       "reviewCategory": "libraries",
       "versionPolicyName": "rush"
     },
+    {
+      "packageName": "@rushstack/rush-sdk",
+      "projectFolder": "apps/rush-sdk",
+      "reviewCategory": "libraries",
+      "versionPolicyName": "rush"
+    },
```

Comment:
We decided to stick `rush-lib` under `apps` so it would be next to `rush`. Should this go under `apps` too, or should we consider moving the non-app projects out of `apps`?

Reply:
HBO's monorepo got into this debate recently also, basically "what constitutes an 'app'?" as far as the category folders go.

Eventually we came to the opinion that:
- "Apps" are experiences that a human launches or interacts with directly.
- The main value of the "apps" category folder is to help newcomers to understand "What apps are developed in this monorepo?"

Under this reasoning, `rush-sdk` and `rush-lib` and should move under the "libraries" category.  (If we rename `rush-lib` to `rush-engine` one day, then it would be a purely internal component, but would still make sense to go under "libraries" I think.)

If we move `rush-lib` that should be a separate PR.  But I'll move `rush-sdk`. 👍

Reply:
Done